### PR TITLE
Docs: Fixes to image guide. [skip ci]

### PR DESCRIPTION
**Pull request by godai78**
CKEditor 5 image feature and image alignment docs - fixes and minor changes. Scope: guide, snippets.  
  
Still missing: live example for defining custom styles.

### CODE REVIEW

**Comment by AnnaTomanek** [comment]
It looks like the image resize API docs have never been reviewed before :wink: 

@godai78 can you have a look and correct the API docs comments in these files (ckeditor5-image/src/imageresize)

**Comment by AnnaTomanek** [comment]
Snippets: I'm fine with using the "meta" content inside, at least for the time being until we figure out something more interesting. However, when reviewing the snippets, I noticed that at some point, when adding the resize and link feature, we deviated a bit, and for consistency reasons, a few things could be changed:

*   [ ] There is no reason to use `<h3>` elements inside, `<p>` was fine.
*   [ ] As a rule, we should not have a situation where the only instructions are inside the editor and not in the guide text. For all features except for resize and link this was done correctly, but now we are missing this sentence or two in the description that would serve as an introduction or instruction to the demo.

File: packages/ckeditor5-image/docs/_snippets/features/image-style-presentational.html [hunk 1]
```diff
@@ -11,5 +11,5 @@
 		<img src="%BASE_PATH%/assets/img/fields.jpg" alt="Autumn fields">
 	</figure>
 
-	<p>Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here.</p>
+	<p>Yet another presentation paragraph and lots of text here, again. Quite a lot of text, just like before. And all the same all this amount of text forms a whole long paragraph. This whole lot of text is here to help you see the image's alignment in context. It serves no other purpose, though. But we still find it useful and plain like it. And it is even longer than the previous one. Go on an use the style buttons to make the most out of this pretty text block and the image alignment demo.</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-image/docs/_snippets/features/image-style-presentational.html:14]:
```suggestion
	<p>Yet another sample paragraph and lots of text here, again. Quite a lot of text, just like before. And this huge amount of text forms a whole long paragraph. This whole lot of text is here to help you see the image's alignment in context. It serves no other purpose, though. But we still find it useful and plain like it. And it is even longer than the previous one. Go on an use the image style buttons to make the most out of this pretty text block and the image alignment demo.</p>
```

File: packages/ckeditor5-image/docs/_snippets/features/image-style.html [hunk 2]
```diff
@@ -11,5 +11,5 @@
 		<img src="%BASE_PATH%/assets/img/fields.jpg" alt="Autumn fields">
 	</figure>
 
-	<p>Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here. Lots of text here.</p>
+	<p>Lots of text here. Quite a lot of text, indeed. In fact there is plenty of text beside this image and it forms a whole long paragraph. This whole lot of text is here to help you see the image's alignment in context. It serves no other purpose, though. But we still find it useful and plain like it.</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-image/docs/_snippets/features/image-style.html:14]:
```suggestion
	<p>Lots of text here. Quite a lot of text, indeed. In fact, there is plenty of text beside this image and it forms a whole long paragraph. This whole lot of text is here to help you see the image's alignment in context. It serves no other purpose, though. But we still find it useful and plain like it.</p>
```

**Comment by AnnaTomanek** [comment]
@godai78 You've accidentally committed the `yarn.lock` file which as a rule you should never do. You can revert it by running `git reset stable yarn.lock` on the `image-options` branch and then committing the change to `image-options`. Thanks @pomek :heart:

File: packages/ckeditor5-image/docs/_snippets/features/image-resize-buttons-dropdown.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <div id="snippet-image-resize-buttons-dropdown">
-	<h3>Resize me using the dropdown!</h3>
+	<p>Click me and resize using the pop-up menu dropdown!</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-image/docs/_snippets/features/image-resize-buttons-dropdown.html:2]: Terminology, here and somewhere else: what you call the "pop-up menu" is what is called the "contextual toolbar" in CKEditor 5 world. As a general UI element it's linked to the feature itself (so: in this case, it's the "image contextual toolbar") and it replaces dialog windows of the past.

File: packages/ckeditor5-image/docs/features/image.md [hunk 4]
```diff
@@ -446,6 +481,7 @@ ClassicEditor
 	.then( ... )
 	.catch( ... );
 ```
+Check out the differnce in this live demo below.
```
Comment (AnnaTomanek) [inline packages/ckeditor5-image/docs/features/image.md:484]: Typo.

Comment (godai78) [reply packages/ckeditor5-image/docs/features/image.md]: Pushed.

**Pull request merged**